lib/l10n.js

```diff
@@ -678,15 +868,19 @@ var l10n = {
     },
     prompt_reporting_polling_place: {
         en: 'Where did you vote? Please include the address, city and state.',
+        ko: "디에서 투표하셨나요? 주소, 도시, 주를 포함 해 주세요."
     },
     msg_reporting_followup: {
-        en: 'A journalist may follow up with you for more information on your story.'
+        en: 'A journalist may follow up with you for more information on your story.',
+        ko: "러분의 이야기에 대해 기자가 취재를 요청 할 수도 있습니다"
     },
     msg_thanks_for_using: {
         en: 'Thanks for using HelloVote!',
+        ko: "elloVote를 사용 해 주셔서 감사합니다"
```

Comment:
Missing an H for HelloVote?

Reply:
whoops - thought that was intentional but turns out I just copypaste failed. Thanks